examples/kyma-mock/apis/kyma.yaml

```diff
@@ -0,0 +1,319 @@
+openapi: '3.0.0'
```

Comment:
That is not at all the full kyma API, please consider all three APIs located here: https://github.com/kyma-project/kyma/tree/master/docs/application-connector/docs/assets

Reply:
Done